package.json

```diff
@@ -41,7 +41,7 @@
   "peerDependencies": {
     "cldr-data": ">=25",
     "globalize": "^1.1.0-rc.5 <1.3.0",
-    "webpack": "^2.5.1"
+    "webpack": "^3.0.0"
```

Comment:
>  I thought the trend was to have master being the latest and greatest and patches backported to "old" branches ?

It could be as you suggest actually... @jbellenger, any comments?

PS: webpack is moving fast. The first 2.x stable version was in Jan 2017, and 3.x in Jun 2017. 😵